src/icons/index.js

```diff
@@ -0,0 +1,7 @@
+import Logo from './Logo'
+import IconEdit from './IconEdit'
+import IconTrash from './IconTrash'
+
+export { Logo }
```

Comment:
why not export under one object?

Reply:
It's more flexible because you can import them individually or as an object.  I think it's more about taste in this case...

Reply:
I mean why not 
```
{
  Logo,
  IconEdit,
  IconTrash
}
```
?

Reply:
Oh, right. as an object is indeed way better!